assets/common.sh

```diff
@@ -87,6 +87,18 @@ prepare_repository() {
   git reset --hard FETCH_HEAD
 }
 
+sorted_tags() {
+  # return only those tags that are recognised by semver as versions
+  local branch="$1"
+  NODE_PATH=/usr/lib/node_modules node -e '
+    var semver=require("semver");
+    var versions=process.argv.slice(1);
+    versions.filter(semver.valid)
+      .sort(function(a, b) {return semver.compare(a, b);})
```

Comment:
Convert to arrow functions
```suggestion
      .sort((a, b) =>  semver.compare(a, b))
```